### PR TITLE
Neume fusion

### PR DESCRIPTION
**Pull request by henryso**
This is a pretty huge pull request that covers #679, #687, and #692.

In addition to neume fusion and specifically the items requested in those issues, I also cleaned up a few things along the way:
* I removed `L_AUCTA`, instead using `L_AUCTUS_ASCENDENS`
* I made the oriscus shapes consistent in greciliae
* I made the scandicus shapes with a second ambitus of one more consistent
* I cleaned up squarize.py:
  * I turned the widths dictionary back into a global variable for simplicity; this script will not be run from a bigger script anyway, and we use other globals anyway.
  * I took the copyright from the source `.sfd` file, which should make it easier for users (like me) to create custom fonts without changing the script.
* I modified the PDF documentation to take the dimensions directly from the macros.

Please review this, try it out, and let me know if there's anything I missed or that I should change.

### CODE REVIEW

**Comment by eroux** [comment]
Well, that's quite impressive, thanks a lot, this really takes Gregorio one step further! Ok for me to be merged

**Comment by henryso** [comment]
Since @eoux has done the preliminary review, I'll merge it, but due to its size, I'm sure it's not 100% bug-free, so anyone willing to beta test this feature is welcome and encouraged to do so!

File: src/gabc/gabc-glyphs-determination.c [hunk 1]
```diff
@@ -774,31 +881,86 @@ gregorio_glyph *gabc_det_glyphs_from_notes(gregorio_note *current_note,
...
                         liquescentia, current_note->previous);
                 current_glyph_type = G_UNDETERMINED;
             }
-            liquescentia = L_INITIO_DEBILIS;
+            liquescentia = head_liquescentia;
+        }
+
+        if (autofuse) {
+            if (current_glyph_first_note == current_note) {
+                if (first_autofused_note) {
```
Comment (eroux) [inline src/gabc/gabc-glyphs-determination.c]: Testing the oriscus feature, I realize I have a warning here:

```
gabc/gabc-glyphs-determination.c: In function ‘gabc_det_glyphs_from_notes’:
gabc/gabc-glyphs-determination.c:895:20: warning: ‘first_autofused_note’ may be used uninitialized in this function [-Wmaybe-uninitialized]
                 if (first_autofused_note) {
```

(the line number doesn't correspond)

**Comment by henryso** [comment]
That warning is actually a false positive because—in the flow of the loop—autofuse will never be made true without also setting first_autofused_note to true.  My compiler (clang) doesn't generate this warning, but I will put in an initial value to appease your compiler.

**Pull request merged**